xonsh/tools.py

```diff
@@ -1582,9 +1582,13 @@ def expanduser_abs_path(inp):
 def expand_case_matching(s):
```

Comment:
Aight @laerus this is more or less what I expected ^__^ I'll try and reimplement this with regexps! Thanks for the hint!